### ISSUE TITLE
ActionData didn't return a Validation object. Make sure you return { form } from form actions.

### ISSUE THREAD

**Issue by ElijahJohnson5** [ciscoheat/sveltekit-superforms#2]
I am returning a form from my action like so 
```
 const { locals } = event;

  const form = await superValidate(event, schema);

  console.log(form);

  if (!form.valid) {
    return fail(400, { form });
  }
```

with the schema:
```
const schema = z.object({
  firstName: z.string(),
  lastName: z.string(),
  password: z.string(),
  email: z.string().email()
});
```

However I get the error: ActionData didn't return a Validation object. Make sure you return { form } from form actions.
unless I change the fail line to
`return fail(400, { form: { ...form, success: false } })`

I am using the form in my page like so   `const { form, errors } = superForm(data.form);`

Am I missing that I should have to manually set success because the docs don't show that. I am using sveltekit 1.8.3

**Comment by hmiiro** [comment]
Okay, let me be aware of this too as i try this out. Have you tried when
the data you're returning is a list of objects say array of users?

**Comment by ciscoheat** [comment]
Hi, I have just updated the library to version 0.5.4, can you try it out and see if it's fixed? `success` should not exist anymore. 

Also, are you doing a `return { form }` at the end of the successful branch of the function?

**Comment by ciscoheat** [comment]
Found the error now, it should be fixed in version 0.5.5, released in a few moments.

**Comment by ciscoheat** [comment]
Updated now! Now you should be able to remove `success`, thank you for finding this. :)

**Comment by ElijahJohnson5** [comment]
Just tried out the updated version, now its getting mad about `Cannot find module 'sveltekit-flash-message/client'`. When I install that package manually the issue seems to be fixed!

**Comment by ciscoheat** [comment]
Yes, there are some issues right now with integrating the library with sveltekit-flash-message, which are almost impossible to debug without publishing to npm and see what happens. I'll release a new 0.5.x version later when this is fixed.

**Comment by ElijahJohnson5** [comment]
Thanks! You can close this issue then!

**Comment by ciscoheat** [comment]
Ok, just released version 0.5.8, which seems to work fine!
